src/importers/lastpassCsvImporter.ts

```diff
@@ -166,11 +166,23 @@ export class LastPassCsvImporter extends BaseImporter implements Importer {
             if (typeParts.length > 1 && typeParts[0] === 'NoteType' &&
                 (typeParts[1] === 'Credit Card' || typeParts[1] === 'Address')) {
                 if (typeParts[1] === 'Credit Card') {
-                    const mappedData = this.parseSecureNoteMapping<CardView>(extraParts, {
+                    let mappedData = this.parseSecureNoteMapping<CardView>(extraParts, {
                         'Number': 'number',
                         'Name on Card': 'cardholderName',
                         'Security Code': 'code',
+                        'Expiration Date': 'expMonth',
                     });
+
+                    // Split string 'January,2020' into two
+                    const tempExpDate = mappedData[0].expMonth.split(',');
```

Comment:
Can `mappedData[0].expMonth` be null/undefined? Seems you need some null checking here, right?

Reply:
That's right. Done.